src/styles.css

```
@@ -484,6 +484,41 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
+  color: #ffffff !important;
+  background: rgba(255, 255, 255, 0.06);
+  border: 1px solid rgba(255, 255, 255, 0.15);
+  box-shadow: 0 4px 12px rgba(0, 0, 0, 0.25);
+}
+
+.footer-link .material-symbols-outlined {
+  transition: transform 0.25s ease;
+}
+
+.footer-link:hover .material-symbols-outlined {
+  transform: translateX(2px) scale(1.05);
```

Stylelint (17.13.0)

[error] Duplicate property "transform" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)